batch/batch/front_end/templates/openapi.yaml

```diff
@@ -0,0 +1,1449 @@
+          required: true
+          schema:
+            type: string
+        - name: x-hail-return-jinja-context
+          in: header
+          description: "Do not render, and instead return the jinja context json."
+          required: false
+          schema:
+            type: boolean
+            default: false
+            example: true
+      responses:
+        '200':
+          description: "Successfully retrieved the job log."
+          content:
+            application/json:
+              schema:
+                type: object
+  '/billing':
+    get:
+      summary: "Billing page (UI)"
+      description: "Render the billing page, or (dev only) its supporting jinja context"
+      operationId: getBillingUI
+      tags:
+        - UI Support
+      parameters:
+        - name: x-hail-return-jinja-context
+          in: header
+          description: "Do not render, and instead return the jinja context json."
+          required: false
+          schema:
+            type: boolean
+            default: false
+            example: true
+      responses:
+        '200':
+          description: "Successfully retrieved the billing page."
+          content:
+            application/json:
+              schema:
+                type: object
+  '/billing_limits':
+    get:
+      summary: "Billing limits page (UI)"
+      description: "Render the billing limits page, or (dev only) its supporting jinja context"
+      operationId: getBillingLimitsUI
+      tags:
+        - UI Support
+      parameters:
+        - name: x-hail-return-jinja-context
+          in: header
+          description: "Do not render, and instead return the jinja context json."
+          required: false
+          schema:
+            type: boolean
+            default: false
+            example: true
+      responses:
+        '200':
+          description: "Successfully retrieved the billing limits page."
+          content:
+            application/json:
+              schema:
+                type: object
+  '/billing_limits/{billing_project}/edit':
+    post:
+        summary: "Edit billing limits and return rendered result HTML, or (dev only) its supporting jinja context"
+        description: "Edit the billing limits of a billing project."
+        operationId: editBillingLimitsUI
+        tags:
+            - Billing Projects
+        parameters:
+            - name: billing_project
+              in: path
+              description: "The ID of the billing project."
+              required: true
+              schema:
+                  type: string
+            - name: x-hail-return-jinja-context
+              in: header
+              description: "Do not render, and instead return the jinja context json."
+              required: false
+              schema:
+                  type: boolean
+                  default: false
+                  example: true
+        requestBody:
+            content:
+            application/json:
```

Comment:
Adding info from importing this into ZAP. ZAP needs this section to be indented under content:.